tests/unit/expire.tcl

```diff
@@ -261,100 +275,286 @@ start_server {tags {"expire"}} {
+            assert_aof_content $aof {
+                {select *}
+                {set foo1 bar PXAT *}
+                {set foo2 bar PXAT *}
+                {set foo3 bar PXAT *}
+                {set foo4 bar PXAT *}
+                {set foo5 bar PXAT *}
+                {set foo6 bar PXAT *}
+                {set foo7 bar}
+                {pexpireat foo7 *}
+                {set foo8 bar}
+                {pexpireat foo8 *}
+                {set foo9 bar}
+                {pexpireat foo9 *}
+                {set foo10 bar}
+                {pexpireat foo10 *}
+                {set foo11 bar}
+                {del foo11}
+                {set foo12 bar}
+                {pexpireat foo12 *}
+                {set foo13 bar}
+                {pexpireat foo13 *}
+                {set foo14 bar}
+                {pexpireat foo14 *}
+                {set foo15 bar}
+                {pexpireat foo15 *}
+                {set foo16 bar}
+                {restore foo17 * {*} ABSTTL}
+                {restore foo18 * {*} absttl}
+            }
+
+            # Remember the absolute TTLs of all the keys
+            set ttl1 [r expiretime foo1 ptime]
+            set ttl2 [r expiretime foo2 ptime]
+            set ttl3 [r expiretime foo3 ptime]
+            set ttl4 [r expiretime foo4 ptime]
+            set ttl5 [r expiretime foo5 ptime]
+            set ttl6 [r expiretime foo6 ptime]
+            set ttl7 [r expiretime foo7 ptime]
+            set ttl8 [r expiretime foo8 ptime]
+            set ttl9 [r expiretime foo9 ptime]
+            set ttl10 [r expiretime foo10 ptime]
+            assert {[r expiretime foo11 ptime] eq "-2"} ; # foo11 is gone
+            set ttl12 [r expiretime foo12 ptime]
+            set ttl13 [r expiretime foo13 ptime]
+            set ttl14 [r expiretime foo14 ptime]
+            set ttl15 [r expiretime foo15 ptime]
+            assert {[r expiretime foo16 ptime] eq "-1"} ; # foo16 has no TTL
+            set ttl17 [r expiretime foo17 ptime]
+            set ttl18 [r expiretime foo18 ptime]
+
+            # Reload data from AOF
+            r debug loadaof
+
+            # Assert that relative TTLs are roughly the same
+            assert_range [r ttl foo1] 90 98
+            assert_range [r ttl foo2] 90 98
+            assert_range [r ttl foo3] 90 98
+            assert_range [r ttl foo4] 90 98
+            assert_range [r ttl foo5] 90 98
+            assert_range [r ttl foo6] 90 98
+            assert_range [r ttl foo7] 90 98
+            assert_range [r ttl foo8] 90 98
+            assert_range [r ttl foo9] 90 98
+            assert_range [r ttl foo10] 90 98
+            assert {[r ttl foo11] eq "-2"} ; # foo11 is gone
+            assert_range [r ttl foo12] 90 98
+            assert_range [r ttl foo13] 90 98
+            assert_range [r ttl foo14] 90 98
+            assert_range [r ttl foo15] 90 98
+            assert {[r ttl foo16] eq "-1"} ; # foo16 has no TTL
```

Comment:
please use `assert_euqal` here and below instead of `assert`.
the benefit is that when it fails, we can see the values.

Reply:
Thanks for the info. I didn't know the difference. Done.